EmbeddedSocialClient/sdk/src/main/java/com/microsoft/embeddedsocial/ui/fragment/FeedViewMenuListenerFragment.java

```diff
@@ -20,9 +19,9 @@
     private final FeedViewMenuModule menuModule = new FeedViewMenuModule(this);
 
     public FeedViewMenuListenerFragment() {
-        addThemeToMerge(R.style.EmbeddedSocialSdkAppTheme_LightBase);
         Options options = GlobalObjectRegistry.getObject(Options.class);
-        if (options != null && options.showGalleryView()) {
+        addThemeToMerge(options.getBaseTheme());
```

Comment:
previously you were checking if options is null. Is there a danger that GlobalObjectRegistry.getObject will return null? If so, do you need to check if options is not null before calling options.getBaseTheme?

Reply:
I don't believe this danger exists.  The GlobalObjectRegistry is a wrapper around a map which only exposes add and get operations.  The Options item is initialized and placed into this map in the SDK's init function.  The map should always be in scope since it is a static object.  Barring some strange Android app lifecycle corner case, this should never return null.